lib/ui/semantics.dart

```diff
@@ -662,10 +662,10 @@ abstract class StringAttribute extends NativeFieldWrapperClass1 {
     required this.range,
   });
 
-  // The range of the text to which this attribute applies.
+  /// The range of the text to which this attribute applies.
   final TextRange range;
 
-  // Returns a copy of this atttribute with the given range.
+  /// Returns a copy of this atttribute with the given range.
```

Comment:
`//` is appropriate here since as written this is not adequate documentation.

Reply:
Then we need to update the text, because this fails analysis requiring public members be documented.

Reply:
@chunhtai @goderbauer - I've updated the text here, PTAL to make sure it makes sense.